OpenMP.make

```diff
@@ -52,6 +52,12 @@ OMP_NEC_CPU = -fopenmp
 OMP_CRAY_NVIDIA  = -DOMP_TARGET_GPU
 OMP_CLANG_NVIDIA = -DOMP_TARGET_GPU -fopenmp=libomp -fopenmp-targets=nvptx64-nvidia-cuda
 
+# to override AOMP_GPU: $ export AOMP_GPU=gfx906
+AOMP_GPU ?= gfx900
```

Comment:
Does AOMP support more than just gfx900? Can I create a fat binary that contains march code for all attached GPUs?

For instance, run something like `rocm_agent_enumerator | grep gfx9 | grep -v gfx000 | uniq` (if rocm_agent_enumerator exists) and concatenate the resulting list together (e.g. gfx900,gfx906)? Requiring users to come in and change a random line to support their GPU is somewhat user-hostile, IMO.

Reply:
at some point in the near future, we will fix a bug that allows aomp to create fat binaries with more than one gfx.
we do support gfx803 up to gfx90a at the moment. For now , we require users to tell us the gpu target.

Reply:
Then I would still recommend trying to find at least the first GPU in the system automatically. Having to call out gfx targets is a pretty painful Makefile design, IMO. Or perhaps pass it in through a new environment variable at a minimum -- e.g. everything else here (compiler, target) is controlled from outside instead of modifying the file directly.